docs/docs/references/users/README.md

```diff
@@ -1,5 +1,95 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information, more than the name and contact details.
+
+A user profile consists of 3 sections:
+
+- **Basic data**: contains basic info in pre-defined attributes, such as user id, username, email, phone number, and when the user last signed in.
+- **Custom data**: contains additional info in customizable attributes, such as user-preferred color and language.
+- **Identities**: contains the user info retrieved from social sign-in, such as Facebook, GitHub, and WeChat.
+
+### All attributes
+
+The following attributes (except password-related) are visible on the user profile.
+
+:::note The meanings of some following table columns
+
+- **Unique**: Ensures the _uniqueness_ of the values entered into an attribute of a DB table.
+- **Required**: Ensures that the values entered an attribute of a DB table can NOT be NULL.
+- **Editable**: Admin users can only edit the _editable_ attributes using Admin Console or Management API.
+- **Searchable**: Admin users can only search the keyword on the _searchable_ attributes using Admin Console or Management API.
```

Comment:
Those are more like product-level features?  BTW role_names should be editable. Just we haven't implemented it yet.

Reply:
removed these two columns, per offline discussion.